TelCo.ColorCoder/ColorCodeManualTestMethod.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace TelCo.ColorCoder
+{
+    internal class ColorCodeManualTestMethod
+    {
+        public static void ColorCodeManual()
+        {
+            Console.WriteLine("Color code Manual");
+            ColorPair colorPair;
+            List<ColorCodeManual> lstofpairColorCode = ColorCoder.ColorCodeManual.GetColorCodeManual(ColorMapHolder.colorMapMajor, ColorMapHolder.colorMapMinor);
+            int increment = 1;
+            int majorIndex = 0;
+            int minorIndex = 0;
+            foreach (var pairColorCode in lstofpairColorCode)
+            {
+                colorPair = new ColorPair();
+                colorPair.majorColor = pairColorCode.majorColor;
+                colorPair.minorColor = pairColorCode.minorColor;
+                Console.WriteLine("[In]Colors: {0}, [Out] PairNumber: {1}", colorPair, pairColorCode.pairNo);
```

Comment:
Separation of the content-formation from the actual printing makes the Manual testable 👍 
However, the printing code is buried inside test code?